backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -196,6 +197,67 @@ func (fs *experimentStore) GetExperiments(ctx context.Context, configType string
 	return experiments, nil
 }
 
+func (fs *experimentStore) GetListView(ctx context.Context) ([]*experimentation.ListViewItem, error) {
```

Comment:
Can we add a TODO here and link to JIRA to make the pagination driven by the server too?

Reply:
Good idea - I opened a GH issue instead since Jira is not publicly available https://github.com/lyft/clutch/issues/530 :)